src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -96,6 +96,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <TargetPlatformDisplayName Condition="'$(TargetPlatformDisplayName)' == ''">$([Microsoft.Build.Utilities.ToolLocationHelper]::GetPlatformSDKDisplayName($(TargetPlatformIdentifier), $(TargetPlatformVersion)))</TargetPlatformDisplayName>
   </PropertyGroup>
 
+  <PropertyGroup>
+      <DebugFileExt Condition="'$(DebugFileExt)' == ''">.pdb</DebugFileExt>
```

Comment:
I'd say to move this into the following `<PropertyGroup>` where other symbol stuff is defined.